docs/agents/sematext-agent/os-metrics.md

```diff
@@ -0,0 +1,69 @@
+title: Operating system metrics
+description: Operating system metrics collected by Sematext Agent.
+
+The agent collects basic system metrics including memory usage, CPU utilization, disk I/O statistics, and more. Most metrics are directly pulled from the _procsfs_ every 10 seconds, although it is possible to alter the collection interval by modifying the `st-agent.yml` configuration file and specifying the period along with the time unit:
+
+```yaml
+# Metric collection interval for OS, network & container metrics
+interval: 20s
```

Comment:
Isn't it 10?  If we use 10 by default I think we should show 10 here so people don't think we use 20.